### PR TITLE
fix, minor grammar. first-class citizen

### PR DESCRIPTION
**Pull request by bambalamm**
## Description of Problem
From typescript-support.md:
`This means you don't need any additional tooling to use TypeScript with Vue - it has a first-class citizen support.`

I'd think that 'It has a [...] support.' does not sound right here.

## Proposed Solution
So either; 1) *It has a first-class citizens' support*, or 2) *It has first-class citizen support* may fit better?
Option 2) technically means that first-class citizens support it. So I'd guess option 1) is the way to go.

## Additional Information
I'm not a native english speaker, so forgive me for wasting your time if I'm wrong altogether.
Cheers!

### CODE REVIEW

File: src/guide/typescript-support.md
Comment (skirtles-code) [inline src/guide/typescript-support.md:7]: I agree that the current wording is incorrect but only option 2 sounds right to me:

> it has first-class citizen support.

Either that or a total rewording.

Comment (bambalamm) [reply src/guide/typescript-support.md:7]: Option 2 sounds alright to me.

**Review by skirtles-code — APPROVED** [review]

**Comment by bambalamm** [comment]
Should I now do anything else or will the cosmos take it from here?

**Comment by skirtles-code** [comment]
@bambalamm The change looks good to me but you'll need to wait for someone with merge rights to review it. Nothing needed from you until then.

**Review by NataliaTepluhina — APPROVED** [review]
@bambalamm thank you for the fix! Happy to be the cosmos moving this PR to merge 🎉

**Pull request merged**
